packages/flutter/test/widgets/widget_inspector_test_utils.dart

```diff
@@ -12,7 +12,9 @@ import 'package:flutter_test/flutter_test.dart';
 class TestWidgetInspectorService extends Object with WidgetInspectorService {
   final Map<String, ServiceExtensionCallback> extensions = <String, ServiceExtensionCallback>{};
 
-  final Map<String, List<Map<Object, Object?>>> eventsDispatched = <String, List<Map<Object, Object?>>>{};
+  final Map<String, Map<String, List<Map<Object, Object?>>>> eventsDispatched =
```

Comment:
I think it might be time for some `typedef`s or wrapper classes here if we're going four collections deep... :)